docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -176,6 +176,23 @@ For each Trino table which maps to BigQuery view there exists a system table whi
 Given a BigQuery view ``customer_view`` you can send query
 ``SELECT * customer_view$view_definition`` to see the SQL which defines view in BigQuery.
 
+.. _bigquery-sql-support:
+
+SQL support
+-----------
+
+The connector provides read and write access to data and metadata in
+a the BigQuery database. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
```

Comment:
Remove.

Reply:
Should I remove CREATE SCHEMA?

Reply:
No, `CREATE SCHEMA` in BigQuery connector has been supported since version 356.

Reply:
@Jessie212 No need to remove ``` * :doc:`/sql/create-table` ``` because it's supported.

Reply:
Sorry, @ebyhr for unnecessarily removing the entire section.